src/api/cpp/cvc5_kind.h

```diff
@@ -1939,12 +1939,16 @@ enum Kind : int32_t
   /**
    * Match expressions.
    * For example, the smt2 syntax match term
-   *   `(match l (((cons h t) h) (nil 0)))`
+   * `(match l (((cons h t) h) (nil 0)))`
    * is represented by the AST
    *
+   * \rst
+   * .. code:: smtlib
```

Comment:
This is technically not smt-lib (there's no MATCH_BIND_CASE and MATCH_CASE in smt-lib, it's denoted differently), but literally how it is represented internally. Not sure if this makes a difference for the docs, though.

Reply:
I guess we could use lisp instead, or use no highlighting at all.